### ISSUE TITLE
K-fold cross validation

### ISSUE THREAD

**Issue by liluhu0** [open-mmlab/mmclassification#560]
How to do K-fold cross validation with MMCV？

**Comment by Ezra-Yu** [comment]
K-fold is not currently supported.

**Comment by liluhu0** [comment]
> K-fold is not currently supported.

Ok, thanks.  And can you give me some advice on how implement the  code by myself?

**Comment by mzr1996** [comment]
Hello, we don't have K-fold cross-validation for now.
I will create a draft for K-fold implementation, any discussion is welcomed to improve it.

**Comment by liluhu0** [comment]
> Hello, we don't have K-fold cross-validation for now. I will create a draft for K-fold implementation, any discussion is welcomed to improve it.

Nice! Thanks for your reply!

**Comment by zhongqiu1245** [comment]
@mzr1996 Hi
Could you "mosic" repeatdataset in k-fold?
Because I have done relevant experiments, in small sample medical datasets, repeatdataset can improve performance.
So could you "combined" them?

**Comment by zhongqiu1245** [comment]
like:
```
    val=dict(
        type='KFoldDataset',
        fold=0,
        num_splits=5,
        inner_type=dict(
              type='KFoldDataset',
              dataset=dict(
                    times=6,
                    type=dataset_type,
                    data_prefix='data/imagenet/train',
                    pipeline=test_pipeline)))
```

**Comment by mzr1996** [comment]
Closed by #563
